src/styles/theme/globalstyles.js

```diff
@@ -23,6 +16,22 @@ h2 {
   font-size: 24px;
 }
 
+ul {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  color: #93918F;
+}
+
+p {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  color: #93918F;
+}
+
+#subtitle {
```

Comment:
I wouldn't recommend styling an element using a global ID if it's not necessary. But I guess this is only a temporary thing 😄